monkey/tests/unit_tests/monkey_island/cc/resources/test_propagation_credentials.py

```diff
@@ -0,0 +1,28 @@
+import json
+
+import pytest
+from tests.common import StubDIContainer
+from tests.monkey_island import PropagationCredentialsRepository
+from tests.unit_tests.monkey_island.conftest import get_url_for_resource
+
+from monkey_island.cc.repository import ICredentialsRepository
+from monkey_island.cc.resources.propagation_credentials import PropagationCredentials
+
+
+@pytest.fixture
+def flask_client(build_flask_client):
+    container = StubDIContainer()
+
+    container.register(ICredentialsRepository, PropagationCredentialsRepository)
+
+    with build_flask_client(container) as flask_client:
+        yield flask_client
+
+
+def test_propagation_credentials_endpoint_get(flask_client):
+    propagation_credentials_url = get_url_for_resource(PropagationCredentials)
+
+    resp = flask_client.get(propagation_credentials_url)
+
+    assert resp.status_code == 200
+    assert len(json.loads(resp.data)["propagation_credentials"]) == 2
```

Comment:
Let's be more specific. Assert that the retrieved value matches the expected value.